Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -70,7 +71,23 @@ final class PaymentViewController: UIViewController {
     return view
   }()
 
-  private let headerBackgroundView = UIView().disabledAutoresizingIntoConstraints()
+  private lazy var payButtonView: ButtonView = {
+    let view = ButtonView()
+    view.delegate = self
+    view.isEnabled = false
+    return view
+  }()
+
+  private lazy var activityIndicator: UIActivityIndicatorView = {
+    let view =  UIActivityIndicatorView(frame: CGRect(x: 0, y: 0, width: 20, height: 20))
+    let barButton = UIBarButtonItem(customView: view)
+    navigationItem.setRightBarButton(barButton, animated: true)
+    return view
+  }()
+
+  private lazy var headerBackgroundView: UIView = {
+    UIView().disabledAutoresizingIntoConstraints()
+  }()
```

Comment:
```suggestion
  private let headerBackgroundView: UIView = UIView().disabledAutoresizingIntoConstraints()
```

I have previously mentioned how lazy loading properties that get loaded on init anyway is not "a pattern" and this is genuine functionality that we are misusing for no good reason other than "most properties are assigned by closure in this file".

Reply:
Following the same pattern for readability and consistency.